src/core/REPL.pm

```diff
@@ -96,10 +96,8 @@ do {
 
         method teardown-line-editor {
             my $err = linenoiseHistorySave($.history-file);
-
-            if $err {
-                note "Couldn't save your history to $.history-file";
-            }
+            return if !$err;
```

Comment:
Personally, I write `if !...` as `unless ...` or as `if not ...` or as `if ! ...`. I tripped up twice reading this PR 'cause I missed the `!` 😄